src/main/java/com/iridium/iridiumteams/listeners/TeamLevelUpListener.java

```diff
@@ -32,6 +32,7 @@ public void onTeamLevelUp(TeamLevelUpEvent<T, U> event) {
         }
 
         if (event.isFirstTimeAsLevel() && event.getLevel() > 1) {
+            if(!iridiumTeams.getConfiguration().giveLevelRewards) return;
```

Comment:
is this config required? they could just set the levelRewards to an empty array no?

Reply:
you would think.